tests/image/Dockerfile

```diff
@@ -79,7 +79,7 @@ EXPOSE 2375
 
 ENV MINIKUBE_HOME /home/master/.minikube
 
-ENV K8S_VERSION=${K8S_VERSION:-1.22.2}
+ENV K8S_VERSION=${K8S_VERSION:-1.21.6}
```

Comment:
Maybe we should use the same k8s version all way long to keep things simpler?

Reply:
1,21,6 it's last 1.21 version available on minikube preloaded archives